getting-started/scaling-out.md

```diff
@@ -190,6 +190,6 @@ node.
 [detach_data_node]: /api#detach_data_node
 [timescaledb_information-data_node]: /api#timescaledb_information-data_node
 [contact]: https://www.timescale.com/contact
-[data-node-authentication]: /getting-started/data-node-authentication
+[data-node-authentication]: /getting-started/setup/data-node-authentication
```

Comment:
Is this really right? I see `data-node-authentication.md` directly under the `/getting-started/` dir. See here: https://github.com/timescale/docs.timescale.com-content/blob/clustering/getting-started/data-node-authentication.md

Reply:
the file directory doesn't affect the url dirs.  Only the `page-index.js` file matters.

Reply:
That said.  You folks need to decide whether you want data-node-authentication to be inside the dir for `getting-started/setup` or `getting-started` and adjust the `page-index.js` file to reflect that.

Reply:
That can be changed in a different PR though.  This one is necessary to prevent broken links.